packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -133,11 +125,11 @@ export default class CallDiagnosticMetrics {
       identifiers.userId = meeting.userId;
       identifiers.deviceId = meeting.deviceUrl;
       identifiers.orgId = meeting.orgId;
-      // @ts-ignore fix type
+      // @ts-ignore
       identifiers.locusUrl = this.webex.internal.services.get('locus');
     }
 
-    if (meeting.locusUrl && meeting.locusInfo.fullState) {
+    if (meeting?.locusUrl && meeting?.locusInfo?.fullState) {
```

Comment:
If we are adding question mark(?), I don't think we need to double-check. The following should be enough,

```suggestion
    if (meeting?.locusInfo?.fullState) {
```